scripts/refresh_spaces_counters.ts

```diff
@@ -0,0 +1,35 @@
+import 'dotenv/config';
+import db from '../src/helpers/mysql';
+
+// Usage: yarn ts-node scripts/refresh_spaces_counters.ts
+async function main() {
+  const spaces = await db.queryAsync(`SELECT id FROM spaces`);
+
+  console.log(`Found ${spaces.length} spaces`);
+
+  for (const i in spaces) {
+    const stats = await db.queryAsync(
+      `SELECT COUNT(voter) as vote_count, COUNT(DISTINCT(proposal)) as proposal_count FROM votes WHERE space = ? GROUP BY space`,
+      [spaces[i].id]
+    );
+
+    for (const stat of stats) {
+      await db.queryAsync(`UPDATE spaces SET vote_count = ?, proposal_count = ? WHERE id = ?`, [
+        stat.vote_count,
+        stat.proposal_count,
+        spaces[i].id
+      ]);
+      console.log(`${i} / ${spaces.length}`);
+    }
```

Comment:
We get only one value in `stats` no (because we get values of one space)?  why we need a `for` loop or `GROUP BY space` ?

I think it can be just
```suggestion
    const stats = await db.queryAsync(
      `SELECT COUNT(voter) as vote_count, COUNT(DISTINCT(proposal)) as proposal_count FROM votes WHERE space = ?`,
      [spaces[i].id]
    );
    const stat = stats[0];
    await db.queryAsync(`UPDATE spaces SET vote_count = ?, proposal_count = ? WHERE id = ?`, [
      stat.vote_count,
      stat.proposal_count,
      spaces[i].id
    ]);
    console.log(`${i} / ${spaces.length}`);
```